lib/response.ex

```diff
@@ -71,6 +49,13 @@ defmodule HTTPServer.Response do
   end
 
   defp format_response_headers(headers) do
-    for {key, val} <- headers, into: "", do: "#{key}: #{val}#{@carriage_return}"
+    for {key, val} <- headers, into: "", do: "#{dasherize(key)}: #{val}#{@carriage_return}"
```

Comment:
technically a key/value pair, for the server thought it is Response Header Name, and Response Header Value.  I like the short variable name here though.

```suggestion
    for {name, val} <- headers, into: "", do: "#{format_header_name(name)}: #{val}#{@carriage_return}"
```

```suggestion
    for {n, v} <- headers, into: "", do: "#{format_header_name(n)}: #{v}#{@carriage_return}"
```